.github/workflows/build-checks.yml

```diff
@@ -14,7 +14,8 @@ jobs:
 
     - name: 'Install Salesforce CLI'
       run: |
-          wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz
+          
+          wget https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-linux-x64.tar.xz
           mkdir ~/sfdx
           tar xJf sfdx-linux-x64.tar.xz -C ~/sfdx --strip-components 1
```

Comment:
As the filename in line 18 has changed to sf-linux-x64.tar.xz, line 19 has to match.

Reply:
@cgijeffolsen Please review the changes. Thank you!